reaktive/src/commonTest/kotlin/com/badoo/reaktive/observable/DistinctUntilChangedTests.kt

```diff
@@ -0,0 +1,53 @@
+package com.badoo.reaktive.utils.arrayqueue
+
+import com.badoo.reaktive.observable.Observable
+import com.badoo.reaktive.observable.distinctUntilChanged
+import com.badoo.reaktive.observable.observableOf
+import com.badoo.reaktive.observable.subscribe
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+
+private data class Question(val answer: Int)
+
+class DistinctUntilChangedTests {
+
+    private val thirteen = Question(13)
+    private val fortyTwo = Question(42)
+
+    @Test
+    fun distinctUntilChanged() {
```

Comment:
Please check naming style for tests in this PR: https://github.com/badoo/Reaktive/pull/25